docs/learn-faq.md

```diff
@@ -46,15 +46,29 @@ of the parachain and the Polkadot network is completely separate from parachain
 economics. Parachains need collators to continue to progress, so it wouldn't be
 unreasonable to see them incentivize collator nodes in some way but it is completely up to parachain implementers.
 
+### What are the transfer fees for Kusama and Polkadot?
+
+It is important to note that the cost of transferring KSM or DOT tokens is dynamic. Currently, the minimum cost of transferring KSM is 0.01 KSM (the base fee), although this can be changed via governance. However, actual transaction fees will vary based on a variety of factors. Specifically, fee calculation follows the following formula:
+
+```
+base_fee + (tx_length * length_fee) + WeightToFee(weight)
+```
+
+Please see the (Substrate page on fee calculation)[https://substrate.dev/docs/en/next/development/module/fees#fee-calculation] for more detailed information.
+
+### What is the minimum amount of KSM I can have in my account?
+
+It is recommended to always ensure that you keep at least 1 KSM in your account in order to avoid the reaping threshold of 0.1 KSM.  If you have less than 0.1 KSM in your account, that account will be "reaped" - it will no longer occupy space on the chain. This is a dust prevention measure, in order to ensure that the chain is not full of accounts with trivial amounts of KSM taking up space. Since the blockchain is copied to every person running a full node, any savings of space provides dramatic benefits in terms of scalability.
```

Comment:
Did you mean to put "at least 0.1 KSM"? The 1 KSM is at 10x odds with the minimum so it looks a little out of place. Also the existential deposit is 0.01, not 0.1.

Reply:
You're right.  I think it was 0.1 on Alexander, but it's 0.01 on Kusama.  I'll update it.

https://github.com/paritytech/polkadot/blob/077fe75f0c9324f6b63e73b8f146aa57e1dbea0c/runtime/kusama/src/lib.rs#L161